generator/generator.yml

```diff
@@ -339,3 +339,11 @@ modules:
       prtCoverStatus:
         type: EnumAsStateSet
 
+# NEC IX Router
+#
+# https://jpn.nec.com/univerge/ix/Manual/MIB/PICO-SMI-MIB.txt
+# https://jpn.nec.com/univerge/ix/Manual/MIB/PICO-SMI-ID-MIB.txt
+# https://jpn.nec.com/univerge/ix/Manual/MIB/PICO-IPSEC-FLOW-MONITOR-MIB.txt
+  nec_ix:
+    walk:
+      - pico-mib
```

Comment:
We generally don't do "Walk everything" entries because a lot of the data is not appropriate for Prometheus monitoring.

For example, this includes things like config tables, `picoLoginSessionIndex`, etc that probably should not be collected.

I would suggest cutting this down to a few specific trees that are useful for monitoring.

Reply:
Removed low need metrics.
Please check.